spec/graphql/schema/object_spec.rb

```diff
@@ -29,6 +29,8 @@
       name_field = new_object_class.fields.find { |f| f.name == "name" }
       assert_equal "The new description", name_field.description
     end
+
+    it "inherits name and description"
```

Comment:
this is empty 🙊

Reply:
😉 

![image](https://user-images.githubusercontent.com/2231765/33838390-6a80315a-de5d-11e7-8fcc-5b20058d9336.png)

Reply:
added, thanks for the reminder!